### DISCUSSION TITLE
How to backup Vemto Applications

### DISCUSSION THREAD

**Discussion by ballesta** [TiagoSilvaPereira/vemto-issues#204] [Q&A] [answered]
Hi Tiago,

How do I **back up** Vemto Applications ?

I understand from your previous mail that my applications are under **%appdata%**.

But under Ubuntu, I am not able to find where is %appdata% folder located.

Best regards
Bernard

**Comment by TiagoSilvaPereira** [comment, marked as answer]
Hi @ballesta,

On ubuntu, it is generally under **~/.config/vemto/storage**

But with the recent Vemto versions, you could just export the projects you want to backup, as now Vemto supports exporting and importing projects from the Home Page.
